app/converters.js

```diff
@@ -1,5 +1,5 @@
 export function extractUsersAndTeams(orgName, reviewers) {
-  const split = reviewers.split(/[,\s]/);
+  const split = reviewers.split(/[\s,]/).filter(Boolean);
```

Comment:
what does this `.filter(Boolean)` do?

Reply:
filters out empty strings and empty entries

Reply:
Interesting but not obvious, maybe add a comment about it?

Reply:
JavaScript is not obvious however this is basic type coercion in JavaScript

Reply:
Created tests here for it: https://github.com/timja/github-comment-ops/pull/52

Reply:
thanks!